packages/react/src/shape.tsx

```diff
@@ -0,0 +1,66 @@
+import * as React from "@hpcc-js/preact-shim";
+
+interface Circle {
+    cx?: string;
+    cy?: string;
+    radius?: number;
+    fill?: string;
+    stroke?: string;
+}
+
+export const Circle: React.FunctionComponent<Circle> = ({
+    cx,
+    cy,
+    radius = 32,
+    fill = "navy",
+    stroke = fill
+}) => <circle cx={cx} cy={cy} r={radius} fill={fill} stroke={stroke} />;
```

Comment:
If we position the shapes by setting their `top` and `left` bounds it should make it easier to position them relative to each other.